ntbs-service/DataMigration/ImportValidator.cs

```diff
@@ -165,9 +165,13 @@ private async Task CleanContactTracingData(Notification notification, PerformCon
                 await _logger.LogNotificationWarning(context, runId, notification.LegacyId, message);
             }
 
-            notification.ClinicalDetails.Notes +=
-                "The following contact tracing values were invalid and were removed from the notification\n"
+            var contactTracingNotes =
+                "The following contact tracing values were invalid and were removed from the notification:\n"
                 + string.Join(", ", lostData);
+
+            notification.ClinicalDetails.Notes = string.IsNullOrWhiteSpace(notification.ClinicalDetails.Notes)
+                ? contactTracingNotes
+                : notification.ClinicalDetails.Notes + ".\n" + contactTracingNotes;
```

Comment:
Should we add a check here to see if the existing `Notes` ends in a full stop already?

Reply:
Yeah that sounds worthwhile. I'll raise another PR for that

Reply:
Raised #1399